db/migrate/20240715072421_add_rent_fix_status.rb

```diff
@@ -0,0 +1,8 @@
+class AddRentFixStatus < ActiveRecord::Migration[7.0]
+  def change
+    change_table :bulk_uploads, bulk: true do |t|
+      t.column :rent_type_fix_status, :string, default: "not_applied"
+    end
+    remove_column :bulk_uploads, :noint_fix_status, :string, default: "not_applied"
+  end
```

Comment:
This is kinda intentionally weird, not updating the table in bulk triggers rubocop and adding t.remove into the change_table creates an irreversible migration (even when the type is given). I feel like maybe there's some obvious piece of syntax I'm missing, otherwise this does what I'd like it to do